gradle.properties

```diff
@@ -27,6 +27,7 @@ debug_avaritia = false
 
 debug_better_with_mods = false
 debug_blood_magic = false
+debug_blood_arsenal = false
```

Comment:
a < m

Reply:
wow. i didnt know that. im hearing about this now for the first time.

Reply:
you are welcome